doc/memory.rst

```diff
@@ -37,23 +37,23 @@ such as numpy arrays.
 A simple example:
 ~~~~~~~~~~~~~~~~~
 
-  First we define the cache directory::
+  First define the cache directory::
 
-    >>> cachedir = 'your_cache_dir_goes_here'
+    >>> cachedir = 'your_cache_location_directory'
```

Comment:
we might call it `location` as well?

Reply:
I'm not sure. Since the example is using a filesystem directory, it makes more sense to have the variable called `cachedir`. Maybe @GaelVaroquaux have a better idea ?